metacat-metadata/src/main/java/com/netflix/metacat/metadata/store/data/converters/ObjectNodeConverter.java

```diff
@@ -0,0 +1,40 @@
+package com.netflix.metacat.metadata.store.data.converters;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.netflix.metacat.common.json.MetacatJson;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+
+/**
+ * Attribute converter for Jackson ObjectNode type.
+ *
+ * @author rveeramacheneni
+ */
+@Slf4j
+@Converter(autoApply = true)
+@SuppressWarnings("PMD")
+public class ObjectNodeConverter implements AttributeConverter<ObjectNode, String> {
+    private final MetacatJson metacatJson;
+
+    /**
+     * Ctor.
+     *
+     * @param metacatJson the Jackson object mapper.
+     */
+    public ObjectNodeConverter(@NonNull final MetacatJson metacatJson) {
+        this.metacatJson = metacatJson;
+    }
+
+    @Override
+    public String convertToDatabaseColumn(final ObjectNode attribute) {
+        return attribute == null ? null : attribute.toString();
+    }
+
+    @Override
+    public ObjectNode convertToEntityAttribute(final String dbData) {
```

Comment:
Would we want error handling in case of invalid json in this converter or just propagate any exception for higher layer to handle?

Reply:
Would perhaps be best to propagate them to the caller so we don't end up storing invalid json in the first place.